tests/core/pyspec/eth2spec/test/capella/sanity/test_blocks.py

```diff
@@ -133,3 +139,69 @@ def test_exit_and_bls_change(spec, state):
     assert not spec.is_fully_withdrawable_validator(validator, balance, current_epoch)
     assert validator.withdrawable_epoch < spec.FAR_FUTURE_EPOCH
     assert spec.is_fully_withdrawable_validator(validator, balance, validator.withdrawable_epoch)
+
+
+def _valid_withdrawl(spec, state):
```

Comment:
```suggestion
def _valid_withdrawal(spec, state):
```

Reply:
and consider it having an 'action' associated with the name -- e.g. `_perform_valid_withdrawal` or something. Currently, this does not make it clear that it modifies `state`

Reply:
changed the function name. Will leave the others to @hwwhww that knows what she's doing much better than what I could do here.